lib/cogctl/actions/relays/create.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Cogctl.Actions.Relays.Create do
+  use Cogctl.Action, "relays create"
+  import Cogctl.Actions.Relays, only: [render: 3]
```

Comment:
Ah, we're sharing code. That's cool, but I having something like a `Cogctl.Actions.Relays.Util` module (or whatever) would be a better place to stick things that are shared across actions.